### PR TITLE
To-Do list __List structure

### PR DESCRIPTION
**Pull request by nilab-khwaja**
Using webpack to make a bundle for all files.
Using CSS for design the HTML element 
Create an index.js file and set an array of some simple to do tasks.
iterate the tasks using a function.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    checkIcon.checked = tasks[i].completed;
+    span.innerHTML = tasks[i].description;
+    dotsIcon.src = 'dots.png';
+    dotsIcon.style.width = '20px';
+    listItem.appendChild(checkIcon);
+    listItem.appendChild(span);
+    listItem.appendChild(dotsIcon);
+    listContainer.appendChild(listItem);
+  }
+};
```
Comment (ShadyShawkat) [inline src/index.js:41]: - [ ] Please make sure that the tasks appear in order of the `index` values for each task. As per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @nilab-khwaja, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (nilab-khwaja) [reply src/index.js:41]: @ShadyShawkat  Hi,
could you please describe it more ,  I just take index for each task. so what are you pointing?

**Review by Reem-lab — APPROVED** [review]
# Hi @nilab-khwaja, 🤩🔥 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Descriptive PR 🤩🔥
- [ ] Great for set up 🚀
- [ ] Following GitHub flow✅
- [ ] Professional README 🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**